appendix-basic65.tex

```diff
@@ -5355,7 +5350,7 @@ \subsection{LOADIFF}
 
 \begin{verbatim}
 convert <myImage.jpg> <myImage.ppm>
-ppmtoilbm -aga <myImage.pbm> > <myImage.iff>
+ppmtoilbm -aga <myImage.ppm> > <myImage.iff>
```

Comment:
Just wanted to confirm, you gave this a try and confirmed it worked? (I've never tried this before myself, so it's unfamiliar turf).

Ah ok, saw a post in discord from ubik that seems to confirm you're right.